api/build.gradle

```diff
@@ -293,6 +293,10 @@ sourceSets {
     runtimeClasspath += generated.output
   }
   test {
+    // Allow unit tests to directly load config files.
+    resources {
+      srcDir "config/"
```

Comment:
Ooh. Will this mean that we'll no longer have to create a `WorkbenchConfigProvider` per test?